client/src/components/addJobForm/index.tsx

```diff
@@ -17,9 +24,27 @@ function JobForm({ handelClose, showModel }: JobProps) {
       description: '',
     },
     validationSchema: jobSchema,
-    onSubmit: (values) => {
-      console.log(values, 'valuessss');
-      formik.resetForm();
+    onSubmit: async (values) => {
+      try {
+        const job = await axios.post(
+          'api/v1/jobs',
+          {
+            title: values.title,
+            budget: values.budget,
+            time: values.time,
+            category: values.category,
+            description: values.description,
+
+          },
```

Comment:
try to console.log values, You may noticed that it is the same object we're creating here